Resources/Prototypes/Entities/Objects/Specific/Janitorial/spray.yml

```diff
@@ -51,9 +50,8 @@
         maxVol: 250
   - type: Spray
     sprayedPrototype: BigVapor
-    transferAmount: 10
     sprayVelocity: 5
-    sprayAliveTime: 1.5
+    sprayAliveTime: 5
```

Comment:
requesting a smidgen of a buff to `sprayDistance`, which is what i apparently fucked up code failing to do the first time around.